appveyor/scripts/sign.ps1

```diff
@@ -0,0 +1,11 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2010-2024 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+param(
+    [string]$ApiToken,
+    [string]$FileToSign
+)
+
+Submit-SigningRequest -ApiToken $ApiToken -InputArtifactPath $FileToSign -OutputArtifactPath $FileToSign -OrganizationId "12147e94-bba9-4fef-b29b-300398e90c5a" -ProjectSlug "NVDA"  -SigningPolicySlug "release_signing_policy" -WaitForCompletion -Force
```

Comment:
remove double space 

```suggestion
Submit-SigningRequest -ApiToken $ApiToken -InputArtifactPath $FileToSign -OutputArtifactPath $FileToSign -OrganizationId "12147e94-bba9-4fef-b29b-300398e90c5a" -ProjectSlug "NVDA" -SigningPolicySlug "release_signing_policy" -WaitForCompletion -Force
```

Reply:
it might be worth making the OrgId an environment variable so people can customize their own signed builds easier cc @XLTechie for thoughts

Reply:
I left the ability for people to use their own authenticode.pfx files for self-signed builds just like before. The OrgID would only be used if someone was using SignPath to sign their own builds.